frontend-thea-aa/src/lib/components/Statistic.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+    let props: {title: String, stat: any} = $props();
+</script>
+
+<div class="flex flex-col pt-3">
+    <p class="text-stone-500 pb-2">{props.title}</p>
```

Comment:
text-stone-500 also not ideal in case new themes, ideally use one of the following: https://www.shadcn-svelte.com/docs/theming#css-variables